internal/commands/push/command.go

```diff
@@ -568,3 +567,9 @@ func deployDraftAndWait(ui terminal.UI, realmClient realm.Client, remote appRemo
 	ui.Print(terminal.NewTextLog("Deployment complete"))
 	return nil
 }
+
+func discardDraft(ui terminal.UI, realmClient realm.Client, remote appRemote, draftID string) {
+	if err := realmClient.DiscardDraft(remote.GroupID, remote.AppID, draftID); err != nil {
+		ui.Print(terminal.NewWarningLog("Failed to discard the draft created for your deployment"))
+	}
```

Comment:
I'd rather have this inline instead of a void func so there's less 🪄

Reply:
you've beaten me down pretty good on avoiding simple helpers, but I tend to go back to those ways when user-facing messaging is in play (as consistent messaging should be considered a key part of the CLI UX)

fwiw, i envisioned this method potentially needing to scale further with retry logic if this still was an issue for users at some point.  but as a compromise, I can inline the calls for now with hard-coded `var` for making sure the `terminal.NewWarningLog` remains consistent